proposals/1533-host-specific-tasklist.md

```diff
@@ -0,0 +1,241 @@
+# Proposal: Resource Specific Tasklist
+
+Author: Yichao Yang (yycptt)
```

Comment:
@yycptt this is converted from your Google doc with some hand modification

Reply:
The Google doc is somewhat outdated, I'll need to update it. Are you ok if I create another diff to update the proposal after you land?

Reply:
Sounds good. Just want to make sure there's nothing in here that's internal only. I couldn't see anything at a first pass.

Reply:
Just went through the doc again. I don't think it involves any internal information.